keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -84,14 +114,18 @@ and this will implicitly achieve the main goal of
 ## Proposal
 
 - Make the infrastructure generic and simple enough to be easily handed off to the CNCF
-  - Storage buckets (to store the staged/released packages) or anything similar
-  - DNS entries (e.g. apt.kubernetes.io, ...)
-  - package mirror (e.g. a self hosted aptly/artifactory/... or as a service)
-    - have multiple channels, e.g. `stable`, `dev`, `nightly`
+  - Choose a Packages as a Service solution (e.g. Open Build Service...) or build
+    the infrastructure manually. In case we build the infrastructure manually, we need
+    at least:
+    - Storage buckets (to store the staged/released packages) or anything similar
+    - DNS entries (e.g. apt.kubernetes.io, ...)
+    - Package mirror (e.g. a self hosted aptly/artifactory/... or as a service)
+      - have multiple channels, e.g. `stable`, `dev`, `nightly`
 - Run the package builds as part of [krel stage and release]
 - Have a safe way to store the signing key and make it available to the release team and release tooling
```

Comment:
> Have a safe way to store the signing key

It would be great to add detail to why using OBS is safe and how OBS assures safety/security of signing keys.

Reply:
@darix @dirkmueller Can you please provide some feedback on this?

Reply:
signing keys get generted on the signing server and backup'd in encrypted form on the OBS backend servers. There is no way to get the decrypted private part of the key outside of the isolated signing server.

Reply:
> signing keys get generted on the signing server and backup'd in encrypted form on the OBS backend servers. There is no way to get the decrypted private part of the key outside of the isolated signing server.

@adrianschroeter Just to make it clear, so that we can eventually document it, does OBS Admins have access to the private part of the key? Or it's sort of hermetic so only platform itself can access it when needed?

Reply:
OBS Admins have access. But not the average admin in OBS interface, only a reduced set of people who have physical access to the signing server.